src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceImplTest {
+    private static ReportServiceImpl reportService;
+    private static Storage storage;
+
+    @BeforeEach
+    void setUp() {
+        reportService = new ReportServiceImpl();
+        storage = new Storage();
```

Comment:
This two variables can be created just one time. See previous comments about using `BeforeAll` annotation.
You have static methods put() and get() in the Storage class, why we need to create the new instance of Storage?
Please fix all similar cases in all test classes.